torchgeo/datasets/cdl.py

```diff
@@ -136,3 +398,60 @@ def _extract(self) -> None:
         pathname = os.path.join(self.root, self.zipfile_glob)
         for zipfile in glob.iglob(pathname):
             extract_archive(zipfile)
+
+    def plot(  # type: ignore[override]
+        self,
+        sample: Dict[str, Tensor],
```

Comment:
```suggestion
        sample: Dict[str, Any],
```
See my comment on your other PR